src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -781,6 +781,12 @@ hoi4::States CreateStates(const std::map<int, vic3::State>& vic3_states,
          const std::map<int, int> victory_points =
              CreateVictoryPoints(province_set, hoi4_to_vic3_province_mappings, significant_provinces);
 
+         std::set<std::string> cores;
+         if (vic3_state_itr->second.IsIncorporated())
```

Comment:
It's a bool? You should run a test in states that are halfway in the process of incorporation (it takes 2/5/20 years) - are they cores or not by this flag?

Reply:
I'm choosing to make partial incorporation not count.